Makefile

```
@@ -304,4 +304,4 @@ test:
 	skipper make _test
 
 _test:
-	python3 -m pytest discovery-infra/tests --verbose -s
\ No newline at end of file
+	python3 -m pytest $(or ${TEST},${TEST},discovery-infra/tests) --verbose -s
```

maybe add logic to run a specific mark:
```suggestion
	python3 -m $(or ${MARK},${MARK})  $(or ${TEST},${TEST},discovery-infra/tests) --verbose -s
```